telegram/ext/jobqueue.py

```diff
@@ -195,6 +210,46 @@ def run_repeating(self, callback, interval, first=None, context=None, name=None)
         self._put(job, time_spec=first)
         return job
 
+    def run_monthly(self, callback, when, day, context=None, name=None):
+        """Creates a new ``Job`` that runs on a monthly basis and adds it to the queue.
+
+        Args:
+            callback (:obj:`callable`): The callback function that should be executed by the new
+                job. It should take ``bot, job`` as parameters, where ``job`` is the
+                :class:`telegram.ext.Job` instance. It can be used to access its ``Job.context``
+                or change it to a repeating job.
+            when (:obj:`datetime.time`): Time of day at which the job should run. If the timezone
+                (``when.tzinfo``) is ``None``, UTC will be assumed.
+            day (:obj:`int`): Defines the day of the month whereby the job would run. It should
+                be within the range of 1 and 31, inclusive.
+            context (:obj:`object`, optional): Additional data needed for the callback function.
+                Can be accessed through ``job.context`` in the callback. Defaults to ``None``.
+            name (:obj:`str`, optional): The name of the new job. Defaults to
+                ``callback.__name__``.
+
+        Returns:
+            :class:`telegram.ext.Job`: The new ``Job`` instance that has been added to the job
+            queue.
+
+        Notes:
+             This method runs the function every monthly at the given date. This means that
+             if there is no such date within that month, the job would not run.
```

Comment:
Ah, here it is. Still, what do you think about a `strict` argument?

Reply:
yup, i think that's a great idea

Reply:
done!